src/MIT-open-group.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+    <license isOsiApproved="false" licenseId="MIT-open-group"
+             name="MIT Open Group variant">
```

Comment:
As an additional property here between the `name="MIT Open Group variant"` and the `>` closing tag, you'll want to add the property: `listVersionAdded="3.11"`

This is to document that this license was added in the version 3.11 release of the license list.